sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -1011,6 +1011,7 @@ ansiNonReserved
     | PARTITIONS
     | PERCENTLIT
     | PIVOT
+    | PLACING
```

Comment:
This describes this parser change, too. This is more important change than a test suite change.